keyboards/tada68/keymaps/mattgemmell/config.h

```diff
@@ -0,0 +1,3 @@
+#include "../../config.h"
```

Comment:
This should be changed to:

```suggestion
#pragma once
```

The keyboard's `config.h` is automatically included by the build script, so you don't need to include it here.

Reply:
Resolved in latest commit!